TestRunner.py

```diff
@@ -96,24 +96,24 @@ def get_all_transformation_names(heavy=False) -> Iterable:
 
 class FilterRuns(object):
 
-    def __init__(self):
+    def __init__(self, name_of_filter):
         filters = []
         filter_test_cases = []
         package_dir = Path(__file__).resolve()  # --> TestRunner.py
-        filters_dir = package_dir.parent.joinpath("filters")
-        for (_, m, _) in iter_modules([filters_dir]):
```

Comment:
You might still want to keep this for the automated runs -
Like the below command runs all transformations which are not heavy for every PR!
 https://github.com/GEM-benchmark/NL-Augmenter/blob/a5905a08de2f7278ea5fb370bb0fcf4a71c7720a/.github/workflows/nl-action.yml#L30

Reply:
I see. I think there are two options for testing filters: 
1) we add in the test_filter function with a few if clauses to determine the test mode (light, all, etc.) 
2) we can merge the test case for transformation and filter, so there will be only one `test_operation()` function and will determine whether to execute the transformation or filter test cases.

Reply:
I think the second option might be better..

Reply:
Tagging @aadesh11 here so that we don't duplicate anything!